<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>Carol A. Gross Laboratory | UCSF</title>
<link rel="icon" type="image/png" href="img/icons/grossicon.png" />
<link href="css/cag.css" rel="stylesheet" type="text/css" media="all" />
<link href="css/menu.css" rel="stylesheet" type="text/css" media="all" />
<link href="css/people.css" rel="stylesheet" type="text/css" media="all" />
<link href="css/pub.css" rel="stylesheet" type="text/css" media="all" />
<style type="text/css">
@import 'linkedAnchorsPanel.css';
</style>
<script type="text/javascript" src="DOMhelp.js"></script>
<script type="text/javascript" src="innerNav.js"></script>
<meta name="keywords" content="Carol A. Gross, Carol Gross, UCSF, Escherichia coli, E. coli, bacteria, sigma factor, RNA polymerase, sigma-32, sigma32, sigma-E, sigmaE, sigma-70, sigma70, heat shock, cold shock, chemical genetics, promoter, unfolded protein response, stress response, extracytoplasmic stress response, transcription, sigma-core, antisigma, anti-sigma, rpoD, rpoH, rpoE, RseA, RseB, SurA, PPIase, DnaK, DnaJ, GroE, GroEL">
</head>
<body>
<div id="banner"> <img src="img/bannerblack.gif" alt="UCSF navigation bar" width="657" height="30" border="0" align="top" usemap="#ucsf"/>
	<map name="ucsf" id="ucsf">
		<area shape="rect" alt="UCSF home page" coords="26,3,255,30" href="http://www.ucsf.edu/" />
		<area shape="rect" alt="About UCSF" coords="262,6,326,23" href="http://www.ucsf.edu/about_ucsf/" />
		<area shape="rect" alt="Search UCSF" coords="333,4,401,24" href="http://www.ucsf.edu/search/" />
		<area shape="rect" alt="UCSF Medical Center" coords="413,7,519,22" href="http://www.ucsfhealth.org/" />
	</map>
</div>
<div id="mainBlock">
	<div id="titleBlock">
		<div id="upLBox"> <img class="uLimg" src="img/icons/beaker_icon_grad.png" name="ulimage"> </div>
		<div id="title"> <span> </span></div>
	</div>
	<div id = "contentBlock">
		<div id="menu">
			<div id ="aboutButton" class="menuButton"> <a href="index.html" class="menuLink"> </a> </div>
			<div id ="peopleButton" class="menuButton"> <a href="people.html" class="menuLink"> </a> </div>
			<div id ="resButton" class="menuButton"> <a href="research.html" class="menuLink"> </a> </div>
			<div id ="pubsButton" class="menuButton"> <a href="publications.html" class="menuLink"> </a> </div>
		</div>
		<!-- this is where main content goes -->
		<div id = "content">
			<div class="text_block"> Carol Gross's laboratory studies regulatory networks in <span class="org_name">E. coli</span>, an organism that is amenable to genetic and biochemical methods.  Current lab projects focus on three related areas: <br />
			</div>
			<div class="peop_grp_title"> Molecular basis for control of transcription in <span class="org_name">E. coli</span> </div>
			<div class="div_bar"> <span> </span></div>
			
			<div class="text_block"> <br /> Prokaryotes dynamically alter their gene expression programs to suit their environmental circumstance.  The &sigma; subunits of RNA polymerase are central to this adaptation: &sigma;s instigate gene expression by recognizing and melting promoter DNA elements, and each &sigma; represents a specific set of promoters.  We study the molecular basis for promoter melting, and the basis for the promoters specificity of &sigma;s.  We have established specific interactions between residues in alternative &sigma;s and nucleotides in promoters that enable transcription initiation and determine promoter specificity.  <br />
				<br />
				We integrate this knowledge with screens of promoter libraries to model promoter strength and function based on promoter sequence.  Sequenced bacterial genomes cannot be translated into transcriptional regulatory circuit maps largely because accurate prediction of promoters from sequence information is difficult.  We perform in vivo and in vitro activity assays on libraries of natural and mutated forms of <span class="org_name"> E. coli </span> &sigma;E promoters.  We then apply position weight matrix-based models to identify active promoters and predict promoter strength based on promoter sequence. <br />
				<br />
				Representative publications:<br />
				<div class="pub"> <a class="pub_auth" href="http://www.ncbi.nlm.nih.gov/pubmed/19833768">Rhodius VA, Mutalik VK. </a> <span class="pub_yr"> 2010. </span> <span class="pub_ti"> Predicting strength and function for promoters of the Escherichia coli alternative sigma factor, sigmaE. </span> <span class="pub_jrnl"> Proceedings of the National Academy of Sciences, </span> <span class="pub_vol"> 107(7):2854-9 </span> </div>
				<div class="pub"> <a class="pub_auth" href="http://www.ncbi.nlm.nih.gov/pubmed/19833768">Koo BM, Rhodius VA, Nonaka G, deHaseth PL, Gross CA. </a> <span class="pub_yr"> 2009. </span> <span class="pub_ti"> Reduced capacity of alternative sigmas to melt promoters ensures stringent promoter recognition. </span> <span class="pub_jrnl"> Genes &amp; Development, </span> <span class="pub_vol"> 23: 2426:2436. </span> </div>
				<div class="pub"> <a class="pub_auth" href="http://www.ncbi.nlm.nih.gov/pubmed/14988563"> Young BA, Gruber TM, Gross CA. </a> <span class="pub_yr"> 2004. </span> <span class="pub_ti"> Minimal machinery of RNA polymerase holoenzyme sufficient for promoter melting. </span> <span class="pub_jrnl"> Science, </span> <span class="pub_vol"> 03(5662):1382-4. </span> </div>
			</div>
			<div class="peop_grp_title"> <span class="org_name">E. coli</span> stress response circuit architecture</div>
			<div class="div_bar"> <span> </span></div>
			<div class="text_block"> <br />
			 <span class="org_name"> E. coli </span> employs intricate control circuitry to monitor and respond to stresses presented by its environment.  We are defining the molecular mechanisms responsible for activating and modulating the activity of stress responses.  We also define the genome-wide outputs of stress response circuits, allowing us to place the circuits in a global context of gene expression. <br /> <br />

			  The cytoplasmic heat stress response is mediated by &sigma;32.  The &sigma;32 mRNA directly senses changes in temperature and unleashes a chaperone response that sequesters and re-folds heat-denatured proteins.  Our lab has revealed that the &sigma;32 response is dynamically controlled by multiple feedback loops that integrate information from cytoplasmic folding conditions to regulate &sigma;32 activity.  <br /> <br />

			  The periplasmic response, mediated by &sigma;E, responds to unfolded proteins in the periplasm.  Our lab has established circuitry integrating chaperones, proteases, and sRNAs to activate and regulation of the response.  We are currently studying additional inputs and outputs to the reponse, as well as modelling the roles of different parts of this control circuit. <br />
				<br />
				Genomic technology allows us to determine the global outputs of stress responses.  Using transcriptional arrays, we have shown both the &sigma;32 and &sigma;E-mediated response to have a highly targeted response regulon.  We are currently applying high-throughput sequencing tools to determine the molecular mechanisms through which <span class="org_name"> E. coli </span>'s tranlational capacity is altered in response to temperature stress.  <br />
				<br />
				Representative Publications: <br />
				<div class="pub"> <a class="pub_auth" href="http://www.ncbi.nlm.nih.gov/pubmed/17210793"> Chaba R, Grigorova IL, Flynn JM, Baker TA, Gross CA. </a> <span class="pub_yr"> 2007. </span> <span class="pub_ti"> Design principles of the proteolytic cascade governing the sigmaE-mediated envelope stress response in Escherichia coli: keys to graded, buffered, and rapid signal transduction. </span> <span class="pub_jrnl"> Genes &amp; Development, </span> <span class="pub_vol"> 21(1):124-36. </span> </div>
				<div class="pub"> <a class="pub_auth" href="http://www.ncbi.nlm.nih.gov/pubmed/16336047"> Rhodius VA, Suh WC, Nonaka G, West J, Gross CA. </a> <span class="pub_yr"> 2006. </span> <span class="pub_ti">Conserved and variable functions of the sigmaE stress response in related genomes. </span> <span class="pub_jrnl"> PLoS Biology </span> <span class="pub_vol">(1):e2. </span> </div>
				
			</div>
			<div class="peop_grp_title"> High-throughput, quantitative analyses of genetic interactions in <span class="org_name">E. coli</span><br />
			</div>
			<div class="div_bar"> <span> </span></div>
			<div class="text_block"> <br /> High-throughput genetic interaction studies are a powerful tool for identifying gene function and mapping pathway organization in different genetically tractable organisms. Recent advances in the ability to generate double mutants en masse in <span class="org_name">S. cerevisiae</span> and <span class="org_name">S. pombe</span> have dramatically accelerated the acquisition of genetic interaction information to guide further characterization of these model organisms. <br />
				<br />
				We have recently developed a method, based on F factor driven conjugation, which allows for high-throughput generation of double mutants in <span class="org_name"> E.  coli. </span> This method, termed genetic interaction analysis technology for<span class="org_name"> E. coli </span> (GIANT-<span class="org_name">coli</span>), permits us to systematically generate double-mutant cells on solid media in high-density arrays and assess their fitness. Using GIANT-<span class="org_name">coli </span>, systematic genetic interaction data can be rapidly accumulated and combined with information from small-scale studies as well as with other large datasets (eg protein-protein interaction datasets, gene-expression data) to provide novel insights on bacterial physiology. <br />
				<br />
				We anticipate adapting this methodology to give additional phenotypic readouts such as promoter activity (<span class="org_name">gfp </span>or <span class="org_name">lacZ </span> fusions), biofilm formation (Congo-red or calcofluor plates, crystal violet absorption), siderophore production (chrome azurol S plates), and growth inhibition (halo assays). Moreover, our goal is to extend GIANT-<span class="org_name">coli</span> to other gram-negative and gram-positive bacteria. Together, these methodologies should lead to rapid progress in discovery of gene function and network architecture in bacteria. <br />
				<br />
				Representative publications:<br />
<div class="pub"> <a class="pub_auth" href="http://www.ncbi.nlm.nih.gov/pubmed/19160513"> Typas A, Nichols RJ, Siegele DA, Shales M, Collins SR, Lim B, Braberg H, Yamamoto N, Takeuchi R, Wanner BL, Mori H, Weissman JS, Krogan NJ, Gross CA. </a> <span class="pub_yr"> 2008. </span> <span class="pub_ti"> High-throughput, quantitative analyses of genetic interactions in E. coli. </span> <span class="pub_jrnl"> Nature Methods, </span> <span class="pub_vol"> 5(9):781-7. </span> </div>
				</p>
			</div>
		</div>
	</div>
	<div id="footBlock"> </div>
</div>
